en/sim_gazebo_classic/gazebo_vehicles.md

```diff
@@ -82,7 +82,7 @@ make px4_sitl gazebo-classic_plane
 make px4_sitl gazebo-classic_plane_catapult
 ```
 
-This model simulates hand/catapult launch, which can be used for [fixed wing takeoff](../flying/fixed_wing_takeoff.md#fixed-wing-takeoff) in position mode, takeoff mode, or missions.
+This model simulates hand/catapult launch, which can be used for [fixed wing takeoff](../flight_modes/takeoff.md#fixed-wing-fw) in position mode, takeoff mode, or missions.
```

Comment:
@sfuhrer Can you do fixed wing takeoff in position mode still? Last version I think you could - i.e. put in position mode, then throw and it would take off.

Reply:
Not tested myself but theoretically yes, should be possible.

Reply:
THanks. Will leave it then.